st2tests/st2tests/fixtures/packs/dummy_pack_schema_with_additional_items_1/config.schema.yaml

```diff
@@ -0,0 +1,24 @@
+---
+  region:
+    type: "string"
+    required: false
+    default: "us-east-1"
+  profiles:
+    type: "array"
+    required: false
+    additionalItems:
```

Comment:
As additionaItems can be a dict or boolean, is it possible to add a test with additionalItems with the true/false behaviour test.

Also as an aside do we state which JSON Schema we use? As I think its renamed from additionalItems to items in more recent schema: https://json-schema.org/understanding-json-schema/reference/array.html#additional-items

Reply:
done